### PR TITLE
Selector: filters -> pseudos

### PR DESCRIPTION
**Pull request by timmywil**
Fixes gh-2073

### CODE REVIEW

**Comment by markelog** [comment]
Ref jquery-migrate [issue](https://github.com/jquery/jquery-migrate/issues/146).

I wonder if we should add tests not only for the jQuery constructor for these selectors, but specifically and isolated for these functions

**Comment by dmethvin** [comment]
If we'll start warning about this soon (as in potentially Migrate 3.0) is there a UI ticket for this?

**Comment by timmywil** [comment]
@markelog I'm not sure what you're suggesting. We have tests for the added selectors.

**Comment by markelog** [comment]
@timmywil 

we have stuff like - 
```js
assert.equal( jQuery( "#table td:visible" ).length, 1, "hidden cell is not perceived as visible (#4512). Works on table elements" );
```

but should we have 
```js
  assert.equal(jQuery.expr.pseudos.visible(td), false);
```
Should we have direct tests? Cause tests through jQuery constructor or `jQuery#(find | is)` are proxies, if they work, doesn't mean the `jQuery.expr.pseudos.visible` methods works as expected, maybe jQuery constructor as unindent decorator added something along the way and positive result is just a side-effect.

In other words i'm talking about perfectionism :)

File: src/selector-sizzle.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ define( [
 
 jQuery.find = Sizzle;
 jQuery.expr = Sizzle.selectors;
+
+// Deprecated
 jQuery.expr[ ":" ] = jQuery.expr.pseudos;
```
Comment (gibson042) [inline src/selector-sizzle.js]: Shouldn't this move to the deprecated module?

Comment (timmywil) [inline src/selector-sizzle.js]: I thought it would be odd considering we have selector-native, which doesn't include this.

**Comment by timmywil** [comment]
> if they work, doesn't mean the jQuery.expr.pseudos.visible methods works as expected

I think it does with the tests we have.

**Comment by markelog** [comment]
> I think it does with the tests we have.

Some coverage tools would also agree with me in that assessment, but yeah, i guess we can take care of that when we implement it.

**Comment by timmywil** [comment]
> Some coverage tools would also agree with me in that assessment

How so? It's still hitting the codepaths.

**Pull request closed without merging**